app/src/main/java/ch/epfl/cs311/wanderwave/model/data/TrackRecord.kt

```diff
@@ -0,0 +1,12 @@
+package ch.epfl.cs311.wanderwave.model.data
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "track_records")
+data class TrackRecord(
+    @PrimaryKey(autoGenerate = true) val id: Int = 0, // Unique ID for each record
+    val beaconId: String, // ID of the beacon to which the track was added
+    val trackId: String, // ID of the track added to the beacon
+    val timestamp: Long = System.currentTimeMillis() // Timestamp when the record was created
```

Comment:
Have you tested that this creates the TrackRecord object with the actual current time everytime? 
Using a default value that is not constant seems very cursed. I would remove the default value and pass it when creating it.